mne/evoked.py

```diff
@@ -377,7 +377,7 @@ def plot(self, picks=None, exclude='bads', unit=True, show=True, ylim=None,
         show : bool
             Call pyplot.show() at the end or not.
         ylim : dict
-            ylim for plots. e.g. ylim = dict(eeg=[-200e-6, 200e-6])
+            ylim for plots. e.g. ylim = dict(eeg=[-200, 200])
```

Comment:
Shouldn't that be `-200e-6. 200e-6`?

Also, see line 438 for a similar, but different error :)